src/styles/apply.styl

```diff
@@ -0,0 +1,27 @@
+/* ==========================================================================
+   Apply Section
+   ========================================================================== */
+warning-style() {
+  display: inline-block;
+  padding: 0 5px;
+  border-radius: 2px;
+  color: white;
+}
+.warning {
```

Comment:
If these are more general styles (e.g. for warnings in general), would it be better to move these into its own file? If they're supposed to be tightly coupled to the dashboard, would it be better if the class name reflected that?